### ISSUE TITLE
pretrained weight evaluation

### ISSUE THREAD

**Issue by CircleAAA** [MIT-SPARK/MiDiffusion#1]
good day, would you like to share the pretrained weights for evaluation?

**Comment by SiyiHu** [comment]
This paper is currently under review. We plan to share pretrained weights after we hear back from the reviewers.

**Comment by CircleAAA** [comment]
Wish your paper publish soon. I trained and evaluated the model for bedroom layouts following your project instructions. I stopped training when the epoch loss Oscillated around 0.025. I am not pretty sure if the model is sufficiently trained, so may I ask what was the final epoch loss when you stopped training?

**Comment by SiyiHu** [comment]
The training loss stabilizes to around 0.01. For bedroom, it will take about 40k-45k epochs. For the other two dataset, that will be about 60-65k. You can try training up to 50k (bedroom) and 100k (dining room / living room) as suggested in the config files. The final results should be pretty close to what we reported in the paper. Attached is an example train plot for the bedroom dataset.
![Bedroom_mixed](https://github.com/MIT-SPARK/MiDiffusion/assets/45982924/3b8d5f1c-36a8-403c-ace3-9544d6c01da0)

**Comment by CircleAAA** [comment]
Thank you, I will try 50k epochs first,
